src/js/yeldaWebchatInjector.js

```diff
@@ -158,25 +159,37 @@ class YeldaChat {
         return
       }
 
-      const customImage =  settings.data.image && settings.data.image.url
+      const isVoiceFirstUI = settings.data.hasOwnProperty('isVoiceFirstUI') || false
+      const customImage = settings.data.image && settings.data.image.url
       const hasCustomStyle = settings.data.hasOwnProperty('isDefaultStyle') && !settings.data.isDefaultStyle
 
-      if (!hasCustomStyle || !customImage) {
-        this.webChatContainer.appendChild(this.assistantImage)
-        return
-      }
+      /**
+       * when isVoiceFirstUI is true, instead of adding the assistant image open the chat here,
+       * website-chat will take care rendering the voice first UI
+       */
+      if (!data.isDemo && isVoiceFirstUI) {
+        // voiceFirstUI added to the iframeContainer to remove box-shadow css style
+        // other styles can be added for voice first UI based on this class in the future if needed
+        this.iframeContainer.classList.add('voiceFirstUI')
+        this.openChat()
+      } else {
```

Comment:
don't we need a return here ? if we don't the see the assistant image on mobile, no need to load it, right ?